### DISCUSSION TITLE
Inbound Filters on IP address blocks

### DISCUSSION THREAD

**Discussion by Hydra9268** [getsentry/support#123] [Support] [answered]
The Inbound Filters provide a way to filter IP addresses. 

However, we see an extraordinary number of errors triggered from a 1.1.1.X (actual IP changed). So instead of manually creating 1.1.1.0 through 1.1.1.255. It would be nice if I could block everything from 1.1.1.X.

Can I do this, and if so, how?

**Comment by JLuse** [comment, marked as answer]
Hello, we currently do not support such things as wildcard inputs to block multiple addresses. We do support CIDR ranges however. An example could be 1.1.1.0/24.

**Reply by Hydra9268** [reply]
Perfect! That'll work. Thank you.
